packages/main/src/DatePicker.hbs

```diff
@@ -42,4 +42,6 @@
 	</ui5-input>
 
 	<slot name="formSupport"></slot>
+
+  <slot name="dateRangeDuration"></slot>
```

Comment:
The custom content should be in `DatePickerPopover.hbs` template, and it is in the static area and you can't have a slot there.

I would suggest adding a hook in `DatePickerPopover.hbs`, then you can extend `DateRangePicker.js`, implement the hook, and create some properties that render the text for the buttons.

Reply:
Thanks for updates,
Will try to implement hook for this functionality, might be we will request for dedicated hook for the same in coming days.